saleor/graphql/checkout/types.py

```diff
@@ -116,6 +116,9 @@ class Checkout(CountableDjangoObjectType):
         description="The price of the checkout before shipping, with taxes included.",
     )
     token = graphene.Field(UUID, description=("The checkout's token."), required=True)
+    channel_slug = graphene.String(
+        description=("The checkout's channel slug."), required=True
+    )
```

Comment:
There is `channel` added to `only_fields` below. Why do we need `channel_slug` here?